core/sequencer/metadata/sourceslice.go

```diff
@@ -50,8 +52,8 @@ func New(logID int64, low, high time.Time) (*SourceSlice, error) {
 	}
 	return &SourceSlice{s: &spb.MapMetadata_SourceSlice{
 		LogId:            logID,
-		LowestInclusive:  low.UnixNano(),
-		HighestExclusive: high.UnixNano(),
+		LowestInclusive:  low.UnixNano() / nanosPerQuantum,
+		HighestExclusive: high.UnixNano() / nanosPerQuantum,
```

Comment:
These lines change what's being stored? Does this fix some problem? Please add it to the PR description.

Reply:
Yes, this does indeed fix a problem.  Documented in the description.